interface/main/finder/dynamic_finder.php

```diff
@@ -160,12 +160,15 @@ function persistCriteria(el, e){
 <body class="body_top">
     <div class="<?php echo $container;?> expandable">
         <div class="row">
-            <div class="col-sm-12">
+            <div class="col-sm-10">
                 <h2>
                 <?php echo xlt('Patient Finder') ?> <i id="exp_cont_icon" class="fa <?php echo attr($expand_icon_class);?> oe-superscript-small expand_contract"
                 title="<?php echo attr($expand_title); ?>" aria-hidden="true"></i> <i id="show_hide" class="fa fa-search-plus fa-2x small" title="<?php echo xla('Click to show advanced search'); ?>"></i>
                 </h2>
             </div>
+            <div class="col-sm-2">
+                <button id="create_patient_btn" onclick="top.restoreSession();location='<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Create Patient'); ?></button>
+            </div>
```

Comment:
Should only show this if have following acl:
```
acl_check('patients', 'demo', '', array('write','addonly'))
```

Reply:
Also need to address the above comment.

Reply:
done, still need to work on issue with 2 patients being open at the same time